src/components/side_panel/chart/building_blocks/title/title.ts

```diff
@@ -1,18 +1,60 @@
-import { Component } from "@odoo/owl";
```

Comment:
While you're modifing this file, could you rename it to `chart_title.ts` ? Also rename the .xml and the containing folder. In a separate commit preferably, so we don't haev issues with git losing the file history :)